angular-hub/src/app/pages/discover/index.page.ts

```diff
@@ -63,6 +65,15 @@ export const routeMeta: RouteMeta = {
             <app-event-card [event]="event.attributes"></app-event-card>
           </a>
         </mat-nav-list>
+        <ng-container *ngIf="!currentWeekEvents.length">
+          <p class="mb-4">There are no more events planned this week!</p>
+          <a
+            class="text-xl font-bold bg-primary px-6 py-2 rounded-lg"
+            routerLink="/events"
+            [queryParams]="{ state: 'upcoming' }"
+            >Discover upcoming events</a
+          >
+        </ng-container>
```

Comment:
I thought about it twice and it would be misleading: 'view all' implies not all elements are visible but that's related to the 'upcoming events this week's, not just 'upcoming events.